### DISCUSSION TITLE
“taro组件编译为原生自定义组件”有具体实例吗

### DISCUSSION THREAD

**Discussion by Awen-hub** [NervJS/taro#10850] [Q & A]
[https://nervjs.github.io/taro/docs/taro-in-miniapp#%E6%8A%8A-taro-%E7%BB%84%E4%BB%B6%E7%BC%96%E8%AF%91%E4%B8%BA%E5%8E%9F%E7%94%9F%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BB%84%E4%BB%B6](url)
文档中的描述比较简略，编译出来的产物比较多，不清楚要怎么才能在原生小程序中使用taro编译出来的的“原生自定义组件”，官方有具体的使用案例吗，例如在微信小程序中怎么使用。

**Comment by yanchengqu** [comment]
希望官方提供一些具体实例，按照文档编译出来多出了挺多无用代码

**Reply by Chen-jj** [reply]
https://github.com/NervJS/taro/blob/next/examples/blended-taro-component/README.md
